tests/config.py

```diff
@@ -24,9 +12,9 @@ def get_es_password():
 KIBANA_CONF_DIR = './elk/kibana/'
 LOGSTASH_CONF_DIR = './elk/logstash/'
 
-ES_USER_LOCAL = get_es_usr()
-ES_PASS_LOCAL = get_es_password()
-MAIN_ES_IP = "35.197.82.152"
+ES_USER_LOCAL = ut.get_env("ES_USER")
+ES_PASS_LOCAL = ut.get_env("ES_PASS")
+MAIN_ES_IP = ut.get_env("MAIN_ES_IP")
 MAIN_ES_URL = f"{MAIN_ES_IP}:9200"
```

Comment:
Rather than `MAIN_ES_IP` why not pass in the whole `MAIN_ES_URL` including the port too?